src/main/java/step3/RaceEvaluator.java

```diff
@@ -0,0 +1,22 @@
+package step3;
+
+import java.util.Random;
+
+public class RaceEvaluator {
+
+  private static final int DO_MOVE = 4;
+  private static final int BOUND = 10;
```

Comment:
```suggestion
  private static final int BOUND = 10;
```

의미라는것은 사람마다 다르겠지만, `MAX_~` 이런게 저에게는 좀더 일반화된 이름일것 같긴해네요 😄